market/connector/microsoft365/demo/msGraphDemo/pom.xml

```diff
@@ -38,7 +38,9 @@
           <showErrors>true</showErrors>
           <reportsDirectory>${java.io.tmpdir}</reportsDirectory>
           <properties>
-            <openapi.url>https://raw.githubusercontent.com/microsoftgraph/microsoft-graph-openapi/master/v1.0.json</openapi.url>
+            <openapi.url><![CDATA[
+              https://graphexplorerapi.azurewebsites.net/openapi?tags=me.user,me.calendar,users.calendar&openApiVersion=2&graphVersion=v1.0&format=json
```

Comment:
Why not use openApiVersion=3?

Reply:
with openAPI2 we have support for 'inherited' types and swagger-codegen can reflect that perfectly to Java. 

E.g. the api describes an outlookt event with hierarchy 
graphEntity -> outlookObject -> event

but if we use openapi3 ... all the fields/attributes from parent objects (graphEntity, outlookObject) are not reflected unto the domain model. 

I don't know whether this is an issue how microsoft slices the services, or whether oas2 has better java support. But for now it seems to be the better choice to use swagger2 specs.